### ISSUE TITLE
Fullcalendar not propergating touch events on ionic Project while access from mobile

### ISSUE THREAD

**Issue by Pauldic** [angular-ui/ui-calendar#143]
I tried using fullcalendar in my ionic project everything looks okay but events like dayClick, eventClick dont get propergated while accessed from mobile but from desktop the alert within the event handler function pops up

**Comment by axacheng** [comment]
Thank you Aljosah!

I got same issue and used the workaround that AljosaH offered to fix it.
Below is my fullcalendar bower.conf

{
  "name": "fullcalendar",
  "version": "2.2.3",
  "description": "Full-sized drag & drop event calendar",
  "keywords": [
    "calendar",
    "event",
    "full-sized"
  ],
  "homepage": "http://arshaw.com/fullcalendar/",
  "dependencies": {
    "jquery": ">=1.7.1",
    "moment": ">=2.5.0"
  },
  "devDependencies": {
    "jquery-ui": ">=1.11.1",
    "jquery-simulate-ext": "~1.3.0",
    "jquery-mockjax": "~1.5.4",
    "jasmine-jquery": "~2.0.3",
    "jasmine-fixture": "~1.2.0",
    "moment-timezone": "~0.2.1",
    "bootstrap": "~3.2.0"
  },
  "main": [
    "dist/fullcalendar.js",
    "dist/fullcalendar.css"

**Comment by nespapu** [comment]
@AljosaH Many thanks for the patch! it works very well with my cordova app using fullcalendar v2.3.0 with onsen framework, Thanks again :)

**Comment by wendykurniyanto** [comment]
Hi All
Is there simple example applying FullCalendar in ionic Framework?

Regards,
Wendy

**Comment by larasulfo** [comment]
Hi,
I have similar issue in "version": "1.6.7".  dayClick: function(date, jsEvent, view) is not working when i test mobile device. What is the root of problem?

**Comment by IvanMiiranda** [comment]
@AljosaH The selected day is wrong in the dayClick function with that patch! Fix it please, please =D

**Comment by tacho888** [comment]
Thanks @AljosaH, your solution worked for me. I just had to alter the  `dayMouseClick()` function a little bit because in v2.3 seems that `getCellDate()` function doesn't exists. So the changes are simple:

```javascript
     dayMouseClick: function(ev) {
		// CUSTOM!!!!!!!!!!!
		var _this = this;
		var view = this.view;
		var start; // the inclusive start of the selection
		var dayEl;

		this.coordMap.build();
		var cell = this.coordMap.getCell(ev.pageX, ev.pageY);
		dayEl = _this.getCellDayEl(cell);
		if(dayEl){
			start = moment(dayEl[0].getAttribute('data-date'));
		}
		view.trigger('dayClick', dayEl[0], start, ev);
	},
```

**Comment by YiniYin** [comment]
Have you tried -- data-tap-disabled="true"

**Comment by harishanchu** [comment]
I didn't want to go so deep tuning this library, so I did weird and now touch works. I lost drag functionality but that is okay.

```javascript
		el.on('click', function(ev) {//modified 'mousedown' event to 'click'
			if (
				!$(ev.target).is('.fc-event-container *, .fc-more') && // not an an event element, or "more.." link
				!$(ev.target).closest('.fc-popover').length // not on a popover (like the "more.." events one)
			) {
				_this.dayMousedown(ev);
                $(document).trigger('mouseup'); // manually triggered 'mouseup' event.
			}
		});
```

**Comment by dtelaroli** [comment]
+1

**Comment by leonardobazico** [comment]
+1

**Comment by manj1790** [comment]
use this for touch devices http://fullcalendar.io/blog/2016/03/touch-support/

**Comment by pratikdhaboo** [comment]
When we bundle the app for cordova, dayClick still does not work even with the latest fullcalendar / scheduler.

**Comment by waleedasif322** [comment]
I think this issue can be closed since new releases now support touch events, as @manj1790 has mentioned.

**Comment by lync0056** [comment]
Using the Ionic Framework and the new FullCalendar V2.7  - It appears eventClick is working on native but not dayClick. Anybody else experiencing and have any solution? Thanks

**Comment by pratikdhaboo** [comment]
Yes, I can confirm dayclick not working on native (with cordova) on iOS.

**Comment by bhavna23** [comment]
I need exactly same  IOS calendar features for my webpage app in angular js i.e I need to show all 12 month for current year on single page and then when I select particular month that month should display with events..

**Comment by ketchup208** [comment]
i can also confirm that select is not working with cordova on android
